src/main/java/Snippet.java

```diff
@@ -72,21 +132,32 @@ void setTags(String[] tags) {
     }
 
     Date getCreated() {
-        return created;
+        return createdDate;
     }
 
     Date getModified() {
-        return modified;
+        return modifiedDate;
+    }
+
+    public void setCreatedDate(Date createdDate) {
+        this.createdDate = createdDate;
+    }
+
+    public void setModifiedDate(Date modifiedDate) {
+        this.modifiedDate = modifiedDate;
     }
 
     @Override
     public String toString() {
-        return  "Title: " + title +
-                "\nContent: " + content +
-                "\nlanguage: " + language +
-                "\nTags: " + Arrays.toString(tags) +
-                "\nCreated: " + created +
-                "\nModified: " + modified;
+        return  "\u001B[35mPathToJson: \u001B[0m" + pathToJson +
```

Comment:
Perhaps add this color as a variable "ANSI_PURPLE". Not immediately clear what these characters do.

Reply:
Good observation, maybe it would look even better as applying a method makePurple(String str).